CONTRIBUTING.md

```diff
@@ -1,159 +1,243 @@
 
 ### 1.2 What kinds of contributions are we looking for?
-We love receiving contributions from our community. Any assistance you can provide with regards to bug fixes, feature enhancements, and documentation is more than welcome.
+
+We love receiving contributions from our community. Any assistance you can provide regarding bug fixes, feature enhancements, and documentation is more than welcome.
 
 # 2. Ground Rules
-1. Ensure cross compatibility for Windows, Mac OS and Linux.
+
+1. Ensure cross-compatibility for Windows, Mac OS and Linux.
+
 2. Ensure all Python features used in contributions exist and work in Python 3.8.1 and above.
-3. Create new tests for code you add or bugs you fix. It helps us help you by making sure we don't accidentally break anything :grinning:
-4. Create any issues for new features you'd like to implement and explain why this feature is useful to everyone and not just you personally.
-5. Don't add new cogs unless specifically given approval in an issue discussing said cog idea.
+
+3. Create new tests for code you add or bugs you fix. It helps us help you by making sure we don’t accidentally break anything :grinning:
+
+4. Create any issues for new features you’d like to implement and explain why this feature is useful to everyone and not just you personally.
+
+5. Don’t add new cogs unless specifically given approval in an issue discussing said cog idea.
+
 6. Be welcoming to newcomers and encourage diverse new contributors from all backgrounds. See [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/).
 
 # 3. Your First Contribution
-Unsure of how to get started contributing to Red? Please take a look at the Issues section of this repo and sort by the following labels:
+
+Unsure of how to get started contributing to Red? Please look at the Issues section of this repo and sort by the following labels:
 
 * beginner - issues that can normally be fixed in just a few lines of code and maybe a test or two.
-* help-wanted - issues that are currently unassigned to anyone and may be a bit more involved/complex than issues tagged with beginner.
+
+* help-wanted - issues that are currently unassigned to anyone and maybe a bit more involved/complex than issues tagged with beginner.
 
 **Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
 
-At this point you're ready to start making changes. Feel free to ask for help; everyone was a beginner at some point!
+At this point, you’re ready to make changes. Feel free to ask for help; everyone was a beginner at some point!
```

Comment:
It's an action there for "you are ready to start making making changes" is grammatically correct with a formal undertone.